README.md

```diff
@@ -350,35 +298,29 @@ This architecture consists of the following helm chart installs:
       --set userArtifacts.image=coherence-demo-sidecar:3.0.0-SNAPSHOT \
       coherence/coherence
    ```
-   
-   > **Note:** By default, the latest version of the chart will be used.  You can add `--version 1.0.0` to 
-   > choose a specific version. E.g. 1.0.0 in the above case.
-   
-   You can also choose a specific version of the Coherence Docker image by specifying the following in the
-   above `helm install` command:
-   
+   > **Note:** By default, the latest version of the chart is used. You can add `--version 1.0.0` to select a specific version. For example, 1.0.0 is used in the command.
+
+   You can also choose a specific version of the Coherence Docker image by specifying the following in the `helm install` command:
+
    ```bash
    --set coherence.image=container-registry.oracle.com/middleware/coherence:12.2.1.4.0
    ```
-
    Because we use stateful sets, the coherence cluster will start one pod at a time.
-    
-   Use `kubectl get pods -n sample-coherence-ns` to ensure that all pods are running.
-   All 3 storage-coherence-0/1/2 pods should be running and ready, as below:
+
+   Use `kubectl get pods -n coherence-demo-ns` to ensure that all the pods are running. All three storage-coherence-0/1/2 pods must be running and ready as shown:
```

Comment:
There should only be 1 pod running. 
change to something like
" to ensure that the pod is running.  The pod storage-coherence-0 pod must be running and ready as shown:"